config.default

```diff
@@ -286,11 +286,7 @@ movingAveragePoints = 12
 # compare to observations?
 compareWithObservations = True
 # list of region indices to plot from the region list in [regions] below
-regionIndicesToPlot = [6]
-# Number of points over which to compute moving average (e.g., for monthly
-# output, movingAveragePoints=12 corresponds to a 12-month moving average
-# window)
-movingAveragePoints = 12
+regionIndex = 5
```

Comment:
I think I see a way to move around the rename.  I'm testing now.  

On a more general point.  Given that this routine is only intended to plot nino3.4, which is region 5, I think we could just get rid of the entire timeSeriesNino3.4 section.  Or would it be useful to retain the option for nino 3 and or nino 4?  I'm fine with either.  I would be interested in thoughts from all ( @xylar, @pwolfram, and @milenaveneziani )